docs/DeveloperGuide.md

```diff
@@ -695,22 +726,103 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+--------------------------------------------------------------------------------------------------------------------
+
+### Adding an applicant
+1. Adding an applicant to TrackAScholar while all applicants are displayed in GUI.
+   1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list. The list contains 
+      an applicant named `Alex Yeoh` but does not contain an applicant named `Benjamin low`.
+   
+   1. Test case: 
+      ```
+      add n/Benjamin low 
+      p/98765431 
+      e/benLow@gmail.com 
+      s/NUS Global Merit Scholarship 
+      as/pending 
+      m/Medicine 
+      m/Mathematics
+      ```
+      Expected: An applicant named `Benjamin Low` with the above characteristics
+                will be added to the applicant list and displayed on TrackAScholar GUI.
+      
+   1. Test case:
+      ```
+      add n/Alex Yeoh 
+      p/88712345 
+      e/alexYeoh@gmail.com 
+      s/NUS Merit Scholarship 
+      as/pending 
+      m/Computer Science
+      ```
+      Expected: No changes displayed on TrackAScholar GUI and an error message will be prompted,
+      indicating applicants with duplicate names are not allowed.
+
+   1. Test case:
+      ```
+      add n/Alex Yeoh
```

Comment:
Perhaps consider having the name as `Benjamin Low` to avoid confusing this test case with duplicate applicant name?